website/docs/r/key_vault.html.markdown

```diff
@@ -0,0 +1,167 @@
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurestack_key_vault" "example" {
+  name                        = "examplekeyvault"
+  location                    = azurestack_resource_group.example.location
+  resource_group_name         = azurestack_resource_group.example.name
+  enabled_for_disk_encryption = true
+  tenant_id                   = data.azurestack_client_config.current.tenant_id
+
+  sku_name = "standard"
+
+  access_policy {
+    tenant_id = data.azurestack_client_config.current.tenant_id
+    object_id = data.azurestack_client_config.current.object_id
+
+    key_permissions = [
+      "Get",
+    ]
+
+    secret_permissions = [
+      "Get",
+    ]
+
+    storage_permissions = [
+      "Get",
+    ]
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Key Vault. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the Key Vault. Changing this forces a new resource to be created.
+
+* `sku_name` - (Required) The Name of the SKU used for this Key Vault. Possible values is `standard`.
+
+* `tenant_id` - (Required) The Azure Active Directory tenant ID that should be used for authenticating requests to the key vault.
+
+---
+
+* `access_policy` - (Optional) [A list](/docs/configuration/attr-as-blocks.html) of up to 16 objects describing access policies, as described below.
+
+-> **NOTE** Since `access_policy` can be configured both inline and via the separate `azurestack_key_vault_access_policy` resource, we have to explicitly set it to empty slice (`[]`) to remove it.
+
+* `enabled_for_deployment` - (Optional) Boolean flag to specify whether Azure Virtual Machines are permitted to retrieve certificates stored as secrets from the key vault. Defaults to `false`.
+
+* `enabled_for_disk_encryption` - (Optional) Boolean flag to specify whether Azure Disk Encryption is permitted to retrieve secrets from the vault and unwrap keys. Defaults to `false`.
+
+* `enabled_for_template_deployment` - (Optional) Boolean flag to specify whether Azure Resource Manager is permitted to retrieve secrets from the key vault. Defaults to `false`.
+
+* `enable_rbac_authorization` - (Optional) Boolean flag to specify whether Azure Key Vault uses Role Based Access Control (RBAC) for authorization of data actions. Defaults to `false`.
+
+* `network_acls` - (Optional) A `network_acls` block as defined below.
+
+~> **Note:** This field can only be configured one time and cannot be updated.
+
+<!-- NOTE: not supported -->
+<!-- * `contact` - (Optional) One or more `contact` block as defined below.
+
+~> **Note:** This field can only be set once user has `managecontacts` certificate permission. -->
```

Comment:
even docs nbot supported should be removed

Reply:
https://github.com/hashicorp/terraform-provider-azurestack/pull/151/commits/380ae1ae98407c69f3c3a0ad66a69464ba6b0b2b cleaned